new_platforms/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
```

Comment:
@andschwa I think you previously bumped up the min CMake versions OE used for some feature that you wanted to use, and we settled on 3.10 because that was the VS2017 default?

Reply:
Yes, we need 3.10 to support our our use of OBJECT libraries.